src/main/java/eu/dissco/backend/database/jooq/tables/records/NewUserRecord.java

```diff
@@ -0,0 +1,368 @@
+/*
+ * This file is generated by jOOQ.
+ */
+package eu.dissco.backend.database.jooq.tables.records;
+
+
+import eu.dissco.backend.database.jooq.tables.NewUser;
+
+import java.time.Instant;
+
+import org.jooq.Field;
+import org.jooq.Record1;
+import org.jooq.Record8;
+import org.jooq.Row8;
+import org.jooq.impl.UpdatableRecordImpl;
+
+
+/**
+ * This class is generated by jOOQ.
+ */
+@SuppressWarnings({ "all", "unchecked", "rawtypes" })
+public class NewUserRecord extends UpdatableRecordImpl<NewUserRecord> implements Record8<String, String, String, String, String, String, Instant, Instant> {
```

Comment:
Is this not recursive? NewUserRecord extends Updatable...<**NewUserRecord**>?

Reply:
This code is generated by JOOQ